frontend/src/js/components/SeatSelector.jsx

```diff
@@ -1,26 +1,34 @@
 var React = require('react');
 
 var Router = require('../router.js');
+var Seat = require('./Seat.jsx');
 
-var seats = [{id: 1, selected: false}, {id: 2, selected: false}, {id: 3, selected: false}]; // test data
+var seats = [
+  {id: 1, sectionTitle: 'Permanto', row: '1', seatNumber: '1', selected: false},
+  {id: 2, sectionTitle: 'Permanto', row: '1', seatNumber: '2', selected: false},
+  {id: 3, sectionTitle: 'Permanto', row: '1', seatNumber: '3', selected: false}]; // test data
 
 var SeatSelector = React.createClass({
 
   seatSelected: function (seat) {
     seat.selected = !seat.selected;
+    this.props.onSeatSelected(seats);
     this.forceUpdate();
```

Comment:
As the parent will call setState in this.props.onSeatSelected(), the forceUpdate() is not needed here anymore. Also, I'd be really happy to make it so that everything that changes in the state of the program is stored in one place. In this case it would require getting rid of the seat.selected -property, and maybe storing something like a more general seatStatus = {seat_id: seat_status, ..} in Store's state. BUT this has more to do with the seatmap implementation anyways, I guess.